core/src/test/java/org/hibernate/ogm/backendtck/jpa/JPAConfigWithoutProviderTest.java

```diff
@@ -0,0 +1,186 @@
+		restoreDefaultProviderResolver();
+		testNoProviderConfig();
+	}
+
+	@Test
+	public void testNoProviderConfigWithORMProviderOnly() {
+		forceORMProviderOnly();
+		testNoProviderConfig();
+	}
+
+	@Test
+	public void testNoProviderConfigWithORMProviderOnlyUsingEnableProperty() {
+		forceORMProviderOnly();
+		testNoProviderConfigUsingEnableProperty();
+	}
+
+	@Before
+	public void clearResources() {
+		this.manager = null;
+		this.factory = null;
+	}
+
+	@After
+	public void closeResources() {
+		if ( manager != null ) {
+			manager.close();
+		}
+		if ( factory != null ) {
+			factory.close();
+		}
+	}
+
+	private void testNoProviderConfigUsingEnableProperty() {
+		Map<String, String> defaultTestSettings = TestHelper.getDefaultTestSettings();
+		Map<String, Object> settings = new HashMap<>( defaultTestSettings );
+		factory = Persistence.createEntityManagerFactory( "ogm", settings );
+		manager = factory.createEntityManager();
+		createAndLoadAPoem( manager );
+	}
+
+	private void testNoProviderConfig() {
+		Map<String, String> defaultTestSettings = TestHelper.getDefaultTestSettings();
+
+		// even if hibernate.ogm.enabled isn't set to true
+		// hibernate OGM would be implicitly stated,
+		// considering the absence of some properties,
+		// mandatory for ORM use cases
+		defaultTestSettings.remove( OgmProperties.ENABLED );
+
+		Map<String, Object> settings = new HashMap<>( defaultTestSettings );
+		factory = Persistence.createEntityManagerFactory( "ogm", settings );
+		manager = factory.createEntityManager();
+		createAndLoadAPoem( manager );
+	}
+
+	private void createAndLoadAPoem(EntityManager em) {
+		EntityTransaction trx = em.getTransaction();
+
+		trx.begin();
+		Poem entity = new Poem();
+		entity.setName( POEM_NAME );
+		em.persist( entity );
+
+		Poem poem = em.find( Poem.class, entity.getId() );
+		assertThat( poem.getName() ).isEqualTo( POEM_NAME );
+		trx.commit();
+
+		trx.begin();
+		em.refresh( poem );
+		em.remove( poem );
+		trx.commit();
+	}
+
+	private void forceORMProviderBeforeOGMProvider() {
+		PersistenceProviderResolverHolder.setPersistenceProviderResolver(
+			new FixedOrderPersistenceProviderResolver( new HibernatePersistenceProvider(), new HibernateOgmPersistence() )
+		);
+	}
+
+	private void forceOGMProviderBeforeORMProvider() {
+		PersistenceProviderResolverHolder.setPersistenceProviderResolver(
+			new FixedOrderPersistenceProviderResolver( new HibernateOgmPersistence(), new HibernatePersistenceProvider() )
+		);
+	}
+
+	private void forceORMProviderOnly() {
+		PersistenceProviderResolverHolder.setPersistenceProviderResolver(
+			new FixedOrderPersistenceProviderResolver( new HibernatePersistenceProvider() )
+		);
+	}
+
+	private void restoreDefaultProviderResolver() {
+		// setting null means setting the default one
+		PersistenceProviderResolverHolder.setPersistenceProviderResolver( null );
+	}
+
+	private static class FixedOrderPersistenceProviderResolver implements PersistenceProviderResolver {
```

Comment:
I'm not sure that I've understood well the question. According to the code of `Persistence#createEntityManagerFactory`, the order would seem to matter. Hibernate OGM has to be activated even if the ORM persistence provider precedes the OGM one in the provider list and this hasn't been declared in the `persistence.xml`.

Reply:
I think that the user at API level cannot update the list, it would be a SPI method. On the other hand, the disposition might change, according to behavior of the service loader.